modules/oadp-1-4-4-release-notes.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-4-release-notes_{context}"]
+= {oadp-short} 1.4.4 release notes
+
+{oadp-first} 1.4.4 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers. No code was changed in the product itself compared to that of {oadp-short} 1.4.3.
+
+[id="known-issues-1-4-4_{context}"]
+== Known issues
+
+.Restore operation remains in the `Finalizing` phase for stateful application that uses the `azurefile-csi` storage class
```

Comment:
I suggest to remove this heading and add an asterisk on line no 16. This note is totally redundant to the following know issue. 

If you must keep this heading, consider the following change:

```suggestion
.Issue with restoring an operation 
```